versions/1/changelog.yml

```diff
@@ -10,6 +10,9 @@
   - description: Add tag support for elastic export
     type: bugfix
     link: https://github.com/elastic/package-spec/pull/223
+  - description: Support `ML model` Elasticsearch type.
```

Comment:
nit: please either use ML model or `ml_model` 

`code tags` are reserved for YAML snippets or real manifest properties